ubuntu/common/install_health_checks.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+
+set -e
+
+AZHC_VERSION=v0.1.0
+
+
+pushd /opt/azurehpc/test/
+
+git clone https://github.com/Azure/azurehpc-health-checks.git --branch $AZHC_VERSION
+
+pushd azurehpc-health-checks
+
+NHC_VERSION=1.4.3
+echo "Installed NHC verison $NHC_VERSION"
+
+wget -O nhc-$NHC_VERSION.tar.xz https://github.com/mej/nhc/releases/download/1.4.3/lbnl-nhc-1.4.3.tar.xz
```

Comment:
NIT: pls use ${NHC_VERSION} in 1.4.3/lbnl-nhc-1.4.3.tar.xz

Reply:
will address

Reply:
addressed